index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+      <header id="header">
+        <img id="header-img" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"></img>
+
+        <nav id="nav-bar">
+          <ul>
+            <li><a class="nav-link" href="#section1">Features</a></li>
+            <li><a class="nav-link"  href="#section2">How It Works</a></li>
+            <li><a class="nav-link"  href="#section3">Pricing</a></li>
+          </ul>
+        </nav>
+      </header>
+
+
+
+      <section id="hero">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
+          <input id="submit" type="submit" value="Get Started" class="btn">
+        </form>
+     </section>
+     <table>
+
+      <tr>
+        <section id="section1">
+          <h2>Premium Materials</h2>
+           <p>
+             Our trombones use the shiniest brass which is sourced locally.
+             This will increase the longevity of your purchase.
+           </p>
+        </section>
+      </tr>
+      <tr>
+        <section id="section2">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have
+            finished making it. We also provide free returns if you are not
+            satisfied.
+          </p>
+        </section>
+      </tr>
+      <tr>
+        <section id="section3">
+          <h2>Fast Shipping</h2>
+          <p>
+            We make sure you recieve your trombone as soon as we have
+            finished making it. We also provide free returns if you are not
+            satisfied.
+          </p>
+        </section>
+      </tr>
+    </table>
+
+
+
+
+
+
```

Comment:
In index.html :

Kindly reduce the space between your section to organize your code.
There a too much space between the sections..